application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/common/DurationPercentileTable.java

```diff
@@ -118,37 +113,47 @@ public DurationPercentileTableBuilder() {
 
 		/**
 		 * Adds a data series to this table, corresponding to an event type with a duration
-		 * associated with it. Calling this method adds two columns to the resulting table.
-		 * The first column contains duration values for the event at different percentiles,
-		 * and the second column contains the number of events with duration <= the duration
-		 * at that percentile.
+		 * associated with it. Calling this method adds two columns to the resulting table. The
+		 * first column contains duration values for the event at different percentiles, and the
+		 * second column contains the number of events with duration <= the duration at that
+		 * percentile.
 		 *
-		 * @param durationColId - the ID to be used for the duration column of this series
-		 * @param durationColName - the user-visible name to appear for the duration column header
-		 * @param countColId - the ID to be used for the event count column of this series
-		 * @param countColName - the user-visible name to appear for the event count column header
-		 * @param typeId - the event type ID used to match events belonging to this series
+		 * @param durationColId
+		 *            - the ID to be used for the duration column of this series
+		 * @param durationColName
+		 *            - the user-visible name to appear for the duration column header
+		 * @param countColId
+		 *            - the ID to be used for the event count column of this series
+		 * @param countColName
+		 *            - the user-visible name to appear for the event count column header
+		 * @param typeId
+		 *            - the event type ID used to match events belonging to this series
```

Comment:
Hyphens.

Reply:
Hm. Not sure. But I'll clean up this one.